config/policy.go

```diff
@@ -28,8 +28,9 @@ import (
 type Policy struct {
 	ID string `mapstructure:"-" yaml:"-" json:"-"`
 
-	From string       `mapstructure:"from" yaml:"from"`
-	To   WeightedURLs `mapstructure:"to" yaml:"to"`
+	From     string          `mapstructure:"from" yaml:"from"`
+	To       WeightedURLs    `mapstructure:"to" yaml:"to"`
+	Response *DirectResponse `mapstructure:"response" yaml:"response,omitempty" json:"response,omitempty"`
```

Comment:
Same here, would it make sense to move the `Redirect *PolicyRedirect` field up into this section?